### DISCUSSION TITLE
Required fields when adding a part

### DISCUSSION THREAD

**Discussion by interconnect80** [Part-DB/Part-DB-server#419] [Ideas]
Hi, is it possible to set some part fields as required when adding a part? If not, could that be something added in the future?

 I’d like to use Part-DB, however I need some key fields to always be inputted by the user. Thanks.

**Comment by influx3k** [comment]
Any plans on implementing this?

**Comment by ECCOsea** [comment]
Yea, good thing. I think the best option for add required fields must be set in category of components. Vote +1.

**Comment by influx3k** [comment]
Just wanted to follow up on this.  Something like this would be super useful.  We just had an issue where the Storage Location field was left blank on a couple of items, leading to multiple people hunting around for the them.  If it had been a required field, it could've never of happened.  Any possibility of something like this getting added in the future?

**Comment by influx3k** [comment]
Would still really like to see this implemented.  This would be a very useful feature!
